content/blogs/gui-setup-part2-partitioning.md

```diff
@@ -0,0 +1,104 @@
+---
+title:      "1st-stage GUI setup, Part 2 - October-November 2023: Making partitioning UI work"
+author:     "hbelusca"
+date:       2024-01-23
+tags:       [ newsletter, gui-setup ]
+---
+
+Greetings!
+
+As you may have noticed, I have been quite silent about my work and not writing, on a regular basis, blog posts about what I have done so far.
+Well, I am more concerned about getting _actual_ code written and working before discussing about it,
+instead of doing that about half-done not-yet-tested code, much like what the GUI setup was during these previous months.
+
+This is why it is only now that I am presenting, in this second blog post, the work I made during the months of [October](https://docs.google.com/spreadsheets/d/1Kx80SmSkj1IdomVC9gcbA_MJ7XFiz_YlYYVxoxv-Jgs/view#gid=711109022) and [November](https://docs.google.com/spreadsheets/d/1Kx80SmSkj1IdomVC9gcbA_MJ7XFiz_YlYYVxoxv-Jgs/view#gid=1783614271) 2023, in preparation for the new 1st-stage ReactOS GUI setup.
```

Comment:
```suggestion
That's why the second blog post is out just by now, presenting the work I made during the months of [October](https://docs.google.com/spreadsheets/d/1Kx80SmSkj1IdomVC9gcbA_MJ7XFiz_YlYYVxoxv-Jgs/view#gid=711109022) and [November](https://docs.google.com/spreadsheets/d/1Kx80SmSkj1IdomVC9gcbA_MJ7XFiz_YlYYVxoxv-Jgs/view#gid=1783614271) 2023, in preparation for the new 1st-stage ReactOS GUI setup.
```

Reply:
I'm choosing @cbialorucki's suggestion, that is closer to what I had in mind.